Blueprints/BlueprintDefinitions/Msbuild-NETCore_2_1/CustomRuntimeFunction-FSharp/template/src/BlueprintBaseName.1/Function.fs

```diff
@@ -0,0 +1,36 @@
+namespace BlueprintBaseName._1
+
+open Amazon.Lambda.Core
+open Amazon.Lambda.RuntimeSupport
+open Amazon.Lambda.Serialization.Json
+
+open System
+
+module Function =
+
+
+    /// <summary>
+    /// A simple function that takes a string and does a ToUpper
+    ///
+    /// To use this handler to respond to an AWS event, reference the appropriate package from 
+    /// https://github.com/aws/aws-lambda-dotnet#events
+    /// and change the string input parameter to the desired event type.
+    /// </summary>
+    /// <param name="input"></param>
+    /// <param name="context"></param>
+    /// <returns></returns>
+    let functionHandler (input: string) (_: ILambdaContext) =
+        match input with
+        | null -> String.Empty
+        | _ -> input.ToUpper()
+
+
+    [<EntryPoint>]
+    let main args =
+    
+        let handler = new Func<string, ILambdaContext, string>(functionHandler)
+        use handlerWrapper = HandlerWrapper.GetHandlerWrapper(handler, new JsonSerializer())
+        use bootstrap = new LambdaBootstrap(handlerWrapper)
+   
+        bootstrap.RunAsync().GetAwaiter().GetResult()
+        0
```

Comment:
Slightly more idiomatic suggestion.
- only use `new` on types that are disposable (in conjection with `use`)
- let compiler implicitly convert to C# `Func`
- `_arg` to denote you deliberately are not using them

```fsharp
[<EntryPoint>]
    let main _args =

        use handlerWrapper = HandlerWrapper.GetHandlerWrapper(functionHandler, JsonSerializer())
        use bootstrap = new LambdaBootstrap(handlerWrapper)
   
        bootstrap.RunAsync().GetAwaiter().GetResult()
        0
```

Reply:
Looks like F# compiler is smarter then C# compiler. We had to do the explicit casting of the Func otherwise we got ambiguous overload errors.

Change args to _
Removed "new"